store/copr/coprocessor.go

```diff
@@ -732,7 +735,22 @@ func (worker *copIteratorWorker) handleTaskOnce(bo *tikv.Backoffer, task *copTas
 		worker.logTimeCopTask(costTime, task, bo, resp)
 	}
 	metrics.TiKVCoprocessorHistogram.Observe(costTime.Seconds())
-
+	switch r := resp.Resp.(type) {
+	case *coprocessor.Response:
+		selResp := &tipb.SelectResponse{}
+		proto.Unmarshal(r.Data, selResp)
+		if selResp != nil && len(selResp.GetExecutionSummaries()) > 0 {
+			readExecution := selResp.GetExecutionSummaries()[0]
+			affectRow := int(readExecution.GetNumProducedRows())
+			readTime := float64(readExecution.GetTimeProcessedNs()) / 1000000
+			readByte := r.GetExecDetailsV2().GetScanDetailV2().GetReadBytes()
+			if affectRow < 20 && readByte < 1024*1024 {
+				metrics.TiKVSmallReadDuration.Observe(readTime)
+			} else {
+				metrics.TiKVLargeReadThroughput.Observe(float64(readByte) / (1024 * 1024) / (readTime / 1000))
+			}
+		}
+	}
```

Comment:
How about move those logic to `func (r *selectResult) updateCopRuntimeStats`, to avoid redundantly call `proto.Unmarshal`.